system/src/Grav/Common/Assets.php

```diff
@@ -77,6 +77,7 @@ class Assets
     protected $timestamp = '';
     protected $assets_dir;
     protected $assets_url;
+	protected $attributes;
```

Comment:
From just reading over the code i'm not really seeing where this protected instance variable is used. Below you set the attributes to this value, and then check it for `inline` but why not just do it with the local `$attributes` var?  Is there some persistent need?